core/src/testFixtures/java/org/opensearch/sql/executor/DefaultExecutionEngine.java

```diff
@@ -32,7 +32,7 @@ public void execute(
       while (plan.hasNext()) {
         result.add(plan.next());
       }
-      QueryResponse response = new QueryResponse(new Schema(new ArrayList<>()), new ArrayList<>());
+      QueryResponse response = new QueryResponse(new Schema(new ArrayList<>()), new ArrayList<>(), "");
```

Comment:
Define a new constructor as per https://github.com/Bit-Quill/opensearch-project-sql/pull/237#discussion_r1122570189

Reply:
Added in 5b6f36e94ea33072f3d89b96e70190f45ccb673e